src/core/lib/gprpp/host_port.cc

```diff
@@ -84,19 +89,26 @@ bool SplitHostPort(StringView name, StringView* host, StringView* port) {
   }
   return true;
 }
+}  // namespace
+
+bool SplitHostPort(StringView name, StringView* host, StringView* port) {
+  bool unused;
+  return DoSplitHostPort(name, host, port, &unused);
+}
 
 bool SplitHostPort(StringView name, UniquePtr<char>* host,
                    UniquePtr<char>* port) {
   GPR_DEBUG_ASSERT(host != nullptr && *host == nullptr);
   GPR_DEBUG_ASSERT(port != nullptr && *port == nullptr);
   StringView host_view;
   StringView port_view;
-  const bool ret = SplitHostPort(name, &host_view, &port_view);
+  bool has_port;
+  const bool ret = DoSplitHostPort(name, &host_view, &port_view, &has_port);
   if (ret) {
     // We always set the host, but port is set only when it's non-empty,
```

Comment:
Please update the wording here -- it's not that port is non-empty, it's that there was a ':' in the string optionally followed by a port number.

Reply:
Thank you for noting this.  I updated the comment.